vividus-util/build.gradle

```diff
@@ -10,6 +10,8 @@ dependencies {
     api(group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jdk8')
 
     implementation(group: 'org.apache.commons', name: 'commons-lang3', version: versions.commonsLang3)
+    implementation(group: 'org.apache.parquet', name: 'parquet-avro', version: '1.10.1')
+    implementation(group: 'org.apache.hadoop', name: 'hadoop-common', version: '3.2.0')
```

Comment:
it shouldn't be a part of utils module, since this functionality is very specific